src/write_gcm.c

```diff
@@ -3348,6 +3349,9 @@ static int wg_transmit_unique_segment(const wg_context_t *ctx,
     }
 
     wg_log_json_message(ctx, "Server response:\n%s\n", response);
+    if (strncmp(response, "{}", 2) != 0) {
```

Comment:
Just pointing it out, don't care much either way. Since you're making an assumption that the response is a valid JSON object, might be ok to just add a comment documenting the assumption.